### PR TITLE
feat : e-com landing page

### PR DESCRIPTION
**Pull request by dipteekhd**
Added landing page for e-commerce
https://khd-e-commerce.netlify.app/screens/landing/landing.html

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  z-index: 1;
+}
+.drawer {
+  display: none;
+  border: none;
+  background-color: var(--white);
+}
+@media screen and (min-width: 768px) {
+  .drawer {
+    display: block !important;
```
Comment (shaik-saad) [inline index.css:14]: Vikram Bhaiya has said that it's a good practice to not use !important.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+menuToggle.addEventListener("click", () => {
+  if (menu.style.display || menu.style.display !== "none") {
+    drawer.style.display =
+      !drawer.style.display || drawer.style.display == "none"
+        ? "block"
+        : "none";
+  } else {
+  }
+});
```
Comment (shaik-saad) [inline index.js]: You can remove the else block here, if it's not needed.

**Review by shaik-saad — APPROVED** [review]
Added few suggestions, check them out.

**Review by starver20 — APPROVED** [review]
Other than saad's points it looks good to merge.

File: index.css (hunk 1 above)
Comment (dipteekhd) [reply index.css:14]: added important to override inline style added by JS code.

File: index.js (hunk 2 above)
Comment (dipteekhd) [reply index.js]: Refactored it thank u!

**Pull request merged**